src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -74,6 +74,9 @@ export default class DataTable extends Vue {
   cellWrap = Object.prototype.hasOwnProperty.call(this.$props.config, 'cellWrap')
     ? this.$props.config.cellWrap
     : defaultConfig.cellWrap;
+  showExpandAll = Object.prototype.hasOwnProperty.call(this.$props.config, 'showExpandAll')
+    ? this.$props.config.showExpandAll
+    : defaultConfig.showExpandAll;
```

Comment:
Just curious.. isn't that the same as this?
```suggestion
  showExpandAll = this.$props.config.showExpandAll || defaultConfig.showExpandAll;
```

Reply:
Actually, there is a small difference :wink:

there is no ability to hide `ExpandAll` button in current implementation, because when we pass `showExpandAll === false` as a prop, the default value from the `defaultConfig.showExpandAll` will be assigned. (which is `true`)

`showExpandAll = false || defaultConfig.showExpandAll;`

Reply:
Good catch! Yes, you are right. It cannot be translated like that unless there is some extra comparison, so this is fine as it was. Thanks!